app/services/stats/r205_offender_sar_monthly_performance_report.rb

```diff
@@ -23,7 +23,8 @@ def case_analyzer
     end
 
     def case_scope
-      Case::Base.offender_sar.where(received_date: @period_start..@period_end)
+      number = Case::Base.arel_table[:number]
```

Comment:
I think it might be worth creating a scope on the model for "not_rejected" as we might want to re-use it and naming it will provide some self documentation.

I'm sure what you've done works, but I'm not really familiar with the code you've used.
An alternative would be to use something like
```
where.not("LEFT(cases.number, 1) = 'R'")
```

If we are using the number, then potentially we don't need the state check as well.